src/targets/gpu/kernels/include/migraphx/kernels/dpp.hpp

```diff
@@ -30,11 +30,11 @@
 
 namespace migraphx {
 
-#ifndef MIGRAPHX_HAS_DPP
-#define MIGRAPHX_HAS_DPP 1
+#ifdef MIGRAPHX_USE_HIPRTC
+#define MIGRAPHX_NO_DPP
```

Comment:
This should still use `MIGRAPHX_HAS_DPP` as before and we just pass `-DMIGRAPHX_HAS_DPP=0` when compiling hiprtc.

Reply:
Done.